scripts/bench/benchmarks/pe-class-components/react-dom.production.min.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
Can we avoid checking in built copies of React? People will attempt to edit or download them.